corehq/apps/analytics/README.md

```diff
@@ -0,0 +1,17 @@
+## Analytics
+We use multiple Analytics platforms. Google Analytics and Kissmetrics calls are done mainly using respective Javascript APIs. For Hubspot we use their REST API from server side as well.
```

Comment:
we have server side kiss metrics as well. https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/analytics/tasks.py#L210 It probably doesn't need a full explanation but throwing that link in the examples part could be useful